tests/test_deletes.py

```diff
@@ -0,0 +1,50 @@
+import os
+import shutil
+
+from shallow_backup import destroy_backup_dir
+
+BACKUP_DIR = 'shallow-backup-test-copy-backup-dir'
+TEST_BACKUP_TEXT_FILE = BACKUP_DIR + '/test-file.txt'
```

Comment:
Use `os.path.join` here instead of hardcoding the path. Windows compatibility is in the works.